tests/functional/f/forgotten_debug_statement_py37.py

```diff
@@ -0,0 +1,10 @@
+# pylint: disable=missing-docstring
+
+import sys
+
+breakpoint()  # [forgotten-debug-statement]
+sys.breakpointhook()  # [forgotten-debug-statement]
+b = breakpoint
+b()  # [forgotten-debug-statement]
+bsys = sys.breakpointhook
+bsys()  # [forgotten-debug-statement]
```

Comment:
Just one of those alias checks is already fine. They won't yield a different result for different functions.